Classes/Invite/Providers/AddressBook/BranchInviteTextContactProvider.m

```diff
@@ -123,6 +123,7 @@ - (void)loadContactsFromAddressBookIfPossible:(callbackWithStatus)callback {
         [self loadContactsFromAddressBook:addressBook callback:callback];
     }
     else {
+        CFRelease(addressBook);
```

Comment:
Xcode complained it was leaking.  I added the CFRelease to solve that, but that created another issue, a timing issue with the use of the addressBook.  If you run the code normally, it crashes in the subsequent method that sorts the entries.  

Step through the code, and it works correctly, run it, and it crashes.  A race condition involving the addressBook.  I reverted the code, since a small leak is preferable to a crash.

In the meantime, I got halfway through submitting it (tags created, so I needed to skip a version)

So the net result is that I updated the dependencies, allowing the invite sdk to use the latest version of HMSegmentedController, and got it building with the latest Xcode.